### ISSUE TITLE
Version `2.6.1` is broken

### ISSUE THREAD

**Issue by SebastianKuesters** [marocchino/sticky-pull-request-comment#956]
We get this error when running the action using `marocchino/sticky-pull-request-comment@v2`:

```bash
Error: marocchino/sticky-pull-request-comment/v2/action.yml (Line: 15, Col: 18):
Error: marocchino/sticky-pull-request-comment/v2/action.yml (Line: 16, Col: 14):
Error: marocchino/sticky-pull-request-comment/v2/action.yml (Line: 15, Col: 18): Unrecognized named-value: 'env'. Located at position 1 within expression: env.GITHUB_API_URL
Error: marocchino/sticky-pull-request-comment/v2/action.yml (Line: 16, Col: 14): Unrecognized named-value: 'env'. Located at position 1 within expression: env.GITHUB_API_URL
Error: Fail to load marocchino/sticky-pull-request-comment/v2/action.yml
```

**Comment by ivan-penchev** [comment]
I am getting the same error.

**Comment by marocchino** [comment]
Sorry for that. 🙈  I revert `v2` to `v2.5.0`.

**Comment by ivan-penchev** [comment]
@marocchino thank you for the faster reaction, could confirm that v2 is operational again :)

**Comment by SebastianKuesters** [comment]
@marocchino first of all thanks for maintaining this project.

We are using your action in our pipelines using `marocchino/sticky-pull-request-comment@v2`, so for the next time, it would be great to mark a breaking change by switching to `v3.x.x` instead of `v2.6.x`

**Comment by vearutop** [comment]
Perhaps this "breaking" change wasn't intended, just a slipped bug.

**Comment by marocchino** [comment]
I apologize. It was unintentional that this became a breaking change.
This change will rollback in https://github.com/marocchino/sticky-pull-request-comment/pull/957 .

**Comment by marocchino** [comment]
We have released the rollback as version 2.6.2 and re-tagged it as v2 after testing in [another repository](https://github.com/marocchino/github-action-playground/pull/1). 
I apologize once again for this incidence. Thank you.
